examples/browserify/main.js

```diff
@@ -5,7 +5,7 @@
 
 var pdfjsLib = require('pdfjs-dist');
 
-var pdfPath = '../helloworld/helloworld.pdf';
```

Comment:
It also interesting that this example, and the one below, has been "broken" ever since PR #9904 without anyone noticing (more than a year ago). This would suggest, at least to me, that these examples see very limited use and that cleaning up the fake worker loader code thus isn't a bad idea.